apis/datadoghq/v2alpha1/datadogagent_types.go

```diff
@@ -63,13 +63,13 @@ type DatadogFeatures struct {
 	NPM *NPMFeatureConfig `json:"npm,omitempty"`
 	// USM (Universal Service Monitoring) configuration.
 	USM *USMFeatureConfig `json:"usm,omitempty"`
-	// Dogstatsd configuration.
-	Dogstatsd *DogstatsdConfig `json:"dogstatsd,omitempty"`
+	// DogStatsD configuration.
+	DogStatsD *DogStatsDFeatureConfig `json:"dogStatsD,omitempty"`
```

Comment:
I'd keep the JSON lower case, could be confusing otherwise

Reply:
yea i changed that because the API complained about the difference in casing. i talked to agent core and we decided to change it back to what was originally there (`Dogstatsd / dogstatsd`)